### ISSUE TITLE
Course feedback from the Linux Foundation

### ISSUE THREAD

**Issue by russelltrow** [Green-Software-Foundation/learn#219]
@greenhsu123 I have received feedback from the Linux Foundation about our course:

> I checked the course content on the GSF website and noticed a couple of things that should be updated: the title should be Green Software for Practitioners (instead of Green Software Practitioner) to avoid misleading learners that this course is some sort of certification exam that attests they are practitioners. With this clarifying idea in mind, I have changed the course and the website to say Final Quiz instead of Final Exam - Please update the Welcome and How do you get the best out of this course (last bullet point) sections on the website to replace 'exam' with 'quiz' - learners are not taking a real exam, just a short quiz, so we really want to avoid any misleading statements (we have a lot of exams in our catalog, and the quiz in this course has nothing resembling those exams, therefore we should not really call it an exam).

How would you like to respond to this feedback?

**Comment by russelltrow** [comment]
@NAMRATA-WOKE in today's project call we discussed the Linux Foundation feedback above and agreed to rename the Principles training to "Green Software for Practitioners".

I'm aware that this change might impact more than just the course website. Are you happy to keep the existing name in marketing collateral or should we change everything?

**Comment by NAMRATA-WOKE** [comment]
So its changing from: Green Software Practitioners" to "Green Software for Practitioners" @russelltrow?

**Comment by russelltrow** [comment]
@NAMRATA-WOKE yes, "Green Software Practitioner" to "Green Software for Practitioners". 
What are your thoughts?

**Comment by NAMRATA-WOKE** [comment]
@russelltrow I'm fine with it, the minor adjustment will have to be made across:

- [ ] GSF Website
- [ ] Confluence & Notion
- [ ] Presentation decks
- [ ] GitHub
- [ ] LF training site + certificate of completion 
- [ ] Promotional material for members

**Comment by jawache** [comment]
@russelltrow and @NAMRATA-WOKE this is a very low priority task and given current workload it will have to get done when time becomes free.
